### DISCUSSION TITLE
右手のOLEDが反応しない

### DISCUSSION THREAD

**Discussion by conduc1978** [Yowkees/keyball#323] [Q&A]
４連ピンソケットの破損を乗り越えて、無事に完成しました。
ですが、右手のOLEDが反応しません。左手のOLEDにはファームウェアの書き込みはできました。
左のProMicroにUSBケーブルを入れると、時間はかかりますが、認識されて両手とも使用することができます。
ですが、右のProMicroにUSBケーブルを入れてもファームウェアの書き込みからできません。
これはProMicroの不良を疑ったほうが良いのでしょうか。それとも別の理由があるのでしょうか。
ご教授いただけると助かります。
よろしくお願いします。
![IMG_2238](https://github.com/Yowkees/keyball/assets/142858574/d30f7c19-6d89-4ab5-ba93-ffbae9ea40b2)

**Comment by conduc1978** [comment]
写真がうまく載せれなかったので、もう一度こちらに載せます。
![IMG_2238](https://github.com/Yowkees/keyball/assets/142858574/cd7c179b-4775-40ab-9719-6d5d5b3cb996)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

書き込もうとしているのはRemapのKeyboard catarogで公開しております「Keyball44_via」でしょうか。
ProMicroがUSBデバイスとして認識されない場合、リセットスイッチを２回すばやく押してみてください。
それでもダメでしたらProMicroを取り外して単体で書き込めるかお試しいただければ幸いです。

ご検討お願いいたします。

**Reply by conduc1978** [reply]
ありがとうございます。
リセットスイッチを２回押して、無事に「Keyball44_via」を書き込むことができました。
ですが、OLEDの液晶には何も映りません。
同様に左のOLEDの液晶も映らなくなりました。
これはどういった原因が考えられますか？
ご教授いただけると嬉しいです。

**Comment by Yowkees** [comment]
OLED用の４つのジャンパーと4Pピンソケットの半田付けを見直してみてください。
Keyball44ですと、12Pコンスルーをお使いだと思いますので、逆に上記２点以外、原因となりそうな部分はありませんので、ご確認いただければ幸いです。
